controls-pyqt/exit_program.py

```diff
@@ -0,0 +1,12 @@
+import sys
+
+
+class Exit_Program:
+    def __init__(self, comm: Communications, mcu: MCUInterface):
+        self.comms: Communications = comm
+        self.mcu: MCUInterface = mcu
+        
+    def Exit(self):
+        self.comms.kill_op()
```

Comment:
Would `comms.kill_op()` include `mcu.close_serial()`?

Reply:
It can.